src/main.rs

```diff
@@ -1,23 +1,85 @@
+use std::time::Duration;
+
 use macroquad::prelude::*;
 
 use conways::cell::Cell;
 use conways::grid::Grid;
 
 #[macroquad::main("Conway's Game of Life")]
 async fn main() {
-    clear_background(BLACK);
-
-    let mut grid = Grid::new_from_cells(vec![
-        vec![Cell::Dead, Cell::Alive, Cell::Dead, Cell::Dead],
-        vec![Cell::Alive, Cell::Alive, Cell::Dead, Cell::Dead],
-        vec![Cell::Dead, Cell::Dead, Cell::Alive, Cell::Dead],
-        vec![Cell::Dead, Cell::Dead, Cell::Dead, Cell::Alive],
-    ]); // This is an example grid, for testing purposes only
+    let (rows, cols) = (50, 50);
+    let mut grid = Grid::new(rows, cols);
+    let mut paused = true;
+    let mut show_info = true;
 
     loop {
-        // Here I should draw the grid state on screen I guess.
+        // If it is not paused
+        clear_background(BLACK);
+
+        // Width and Height are defined on every tick cause screen can be resized.
+        let scr_w = screen_width();
+        let scr_h = screen_height();
+        let cell_w = scr_w / grid.cols() as f32;
+        let cell_h = scr_h / grid.rows() as f32;
+
+        // Draw Board
+        #[allow(clippy::needless_range_loop)]
```

Comment:
I think you can remove this `#[allow]`, I tried running clippy without it and it threw no warnings

Reply:
Done!